pkg/runners/builtin_runner/range_check.go

```diff
@@ -0,0 +1,99 @@
+package builtinrunner
+
+import (
+	"errors"
+	"math"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+const INNER_RC_BOUND_SHIFT = 16
+const INNER_RC_BOUND_MASK = math.MaxUint16
+const CELLS_PER_RANGE_CHECK = 1
+
+const N_PARTS = 8
+
+type RangeCheckBuiltinRunner struct {
+	ratio                 *uint32
+	base                  int
+	stopPtr               *int
+	cellsPerInstance      uint32
+	nInputCells           uint32
+	_bound                *lambdaworks.Felt
+	included              bool
+	nParts                uint32
+	instancesPerComponent uint32
```

Comment:
Lets only add the fields we really need here, we wan't to make the vm as simple as possible so we can add them back when we need them